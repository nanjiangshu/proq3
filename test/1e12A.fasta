>1e12A_0001.pdb
LLSSSLWVNVALAGIAILVFVYMGRTIRPGRPRLIWGATLMIPLVSISSYLGLLSGLTVGMIEMPAGHALAGEMVRSQWGRYLTWALSTPMILLALGLLADVDLGSLFTVIAADIGMCVTGLAAAMTTSALLFRWAFYAISCAFFVVVLSALVTDWAASASSAGTAEIFDTLRVLTVVLWLGYPIVWAVGVEGLALVQSVGATSWAYSVLDVFAKYVFAFILLRWVANNERTVAV
