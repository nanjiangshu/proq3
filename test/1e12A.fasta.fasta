>1e12A.fasta
LLSSSLWVNVALAGIAILVFVYMGRTIRPGRPRLIWGATLMIPLVSISSYLGLLSGLTVGMIEMPAGHALAGEMVRSQWGRYLTWALSTPMILLALGLLADVDLGSLFTVIAADIGMCVTGLAAAMTTSALLFRWAFYAISCAFFVVVLSALVTDWAASASSAGTAEIFDTLRVLTVVLWLGYPIVWAVGVEGLALVQSVGATSWAYSVLDVFAKYVFAFILLRWVANNERTVAV
