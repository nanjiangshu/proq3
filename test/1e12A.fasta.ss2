# PSIPRED VFORMAT (PSIPRED V2.5 by David Jones)

   1 L C   0.995  0.006  0.001
   2 L H   0.448  0.622  0.019
   3 S H   0.264  0.599  0.136
   4 S H   0.142  0.846  0.062
   5 S H   0.024  0.881  0.064
   6 L H   0.028  0.864  0.017
   7 W H   0.010  0.959  0.002
   8 V H   0.017  0.971  0.001
   9 N H   0.013  0.984  0.000
  10 V H   0.014  0.984  0.000
  11 A H   0.013  0.989  0.000
  12 L H   0.016  0.985  0.000
  13 A H   0.013  0.991  0.000
  14 G H   0.012  0.991  0.000
  15 I H   0.011  0.992  0.000
  16 A H   0.010  0.993  0.000
  17 I H   0.010  0.993  0.000
  18 L H   0.009  0.993  0.000
  19 V H   0.009  0.993  0.000
  20 F H   0.008  0.994  0.000
  21 V H   0.007  0.992  0.001
  22 Y H   0.010  0.992  0.001
  23 M H   0.022  0.982  0.001
  24 G H   0.115  0.903  0.007
  25 R C   0.683  0.320  0.038
  26 T C   0.792  0.189  0.053
  27 I C   0.814  0.142  0.056
  28 R C   0.868  0.138  0.018
  29 P C   0.887  0.093  0.010
  30 G C   0.588  0.502  0.009
  31 R H   0.477  0.581  0.020
  32 P H   0.228  0.757  0.086
  33 R H   0.126  0.806  0.107
  34 L H   0.009  0.967  0.047
  35 I H   0.001  0.980  0.039
  36 W H   0.001  0.985  0.035
  37 G H   0.002  0.965  0.028
  38 A H   0.007  0.930  0.006
  39 T H   0.015  0.943  0.001
  40 L H   0.012  0.980  0.000
  41 M H   0.013  0.987  0.000
  42 I H   0.012  0.989  0.000
  43 P H   0.015  0.990  0.000
  44 L H   0.014  0.991  0.000
  45 V H   0.013  0.991  0.000
  46 S H   0.013  0.991  0.000
  47 I H   0.012  0.991  0.000
  48 S H   0.013  0.991  0.000
  49 S H   0.015  0.988  0.000
  50 Y H   0.018  0.988  0.000
  51 L H   0.020  0.985  0.000
  52 G H   0.018  0.986  0.000
  53 L H   0.034  0.974  0.001
  54 L H   0.156  0.860  0.003
  55 S H   0.467  0.530  0.005
  56 G C   0.967  0.042  0.005
  57 L C   0.955  0.032  0.015
  58 T C   0.707  0.038  0.218
  59 V E   0.181  0.062  0.640
  60 G E   0.036  0.028  0.925
  61 M E   0.019  0.024  0.939
  62 I E   0.022  0.041  0.900
  63 E E   0.059  0.055  0.845
  64 M E   0.324  0.064  0.470
  65 P C   0.520  0.093  0.314
  66 A C   0.675  0.174  0.143
  67 G C   0.747  0.131  0.128
  68 H C   0.783  0.105  0.112
  69 A C   0.751  0.078  0.192
  70 L C   0.777  0.065  0.161
  71 A C   0.853  0.049  0.078
  72 G C   0.958  0.032  0.020
  73 E C   0.947  0.073  0.018
  74 M C   0.798  0.123  0.067
  75 V C   0.586  0.160  0.133
  76 R C   0.250  0.226  0.227
  77 S E   0.053  0.168  0.454
  78 Q E   0.018  0.204  0.403
  79 W H   0.006  0.727  0.196
  80 G H   0.006  0.954  0.030
  81 R H   0.020  0.885  0.007
  82 Y H   0.060  0.918  0.002
  83 L H   0.095  0.799  0.003
  84 T H   0.223  0.674  0.001
  85 W H   0.047  0.934  0.002
  86 A H   0.021  0.978  0.001
  87 L H   0.039  0.973  0.001
  88 S H   0.055  0.943  0.001
  89 T H   0.033  0.977  0.000
  90 P H   0.022  0.987  0.000
  91 M H   0.017  0.990  0.000
  92 I H   0.014  0.991  0.000
  93 L H   0.014  0.990  0.000
  94 L H   0.017  0.986  0.000
  95 A H   0.017  0.988  0.000
  96 L H   0.017  0.985  0.000
  97 G H   0.015  0.978  0.000
  98 L H   0.022  0.968  0.001
  99 L H   0.090  0.913  0.002
 100 A H   0.490  0.563  0.005
 101 D C   0.964  0.029  0.004
 102 V C   0.979  0.011  0.008
 103 D C   0.960  0.013  0.010
 104 L H   0.088  0.937  0.001
 105 G H   0.044  0.966  0.000
 106 S H   0.017  0.993  0.000
 107 L H   0.009  0.986  0.000
 108 F H   0.010  0.984  0.000
 109 T H   0.013  0.983  0.000
 110 V H   0.014  0.984  0.000
 111 I H   0.013  0.991  0.000
 112 A H   0.015  0.990  0.000
 113 A H   0.016  0.990  0.000
 114 D H   0.023  0.986  0.000
 115 I H   0.013  0.992  0.000
 116 G H   0.013  0.992  0.000
 117 M H   0.012  0.992  0.000
 118 C H   0.011  0.992  0.000
 119 V H   0.012  0.992  0.000
 120 T H   0.016  0.989  0.000
 121 G H   0.044  0.974  0.001
 122 L H   0.039  0.982  0.001
 123 A H   0.062  0.955  0.001
 124 A H   0.033  0.956  0.009
 125 A H   0.063  0.947  0.009
 126 M H   0.111  0.907  0.027
 127 T H   0.297  0.637  0.118
 128 T C   0.715  0.280  0.050
 129 S C   0.872  0.193  0.011
 130 A C   0.879  0.178  0.006
 131 L C   0.588  0.496  0.024
 132 L H   0.350  0.688  0.064
 133 F H   0.105  0.904  0.072
 134 R H   0.009  0.981  0.012
 135 W H   0.007  0.979  0.010
 136 A H   0.009  0.974  0.001
 137 F H   0.008  0.983  0.000
 138 Y H   0.012  0.981  0.000
 139 A H   0.014  0.982  0.000
 140 I H   0.013  0.987  0.000
 141 S H   0.014  0.992  0.000
 142 C H   0.015  0.991  0.000
 143 A H   0.007  0.997  0.000
 144 F H   0.004  0.998  0.000
 145 F H   0.004  0.998  0.000
 146 V H   0.005  0.998  0.000
 147 V H   0.003  0.997  0.000
 148 V H   0.003  0.989  0.007
 149 L H   0.006  0.990  0.005
 150 S H   0.003  0.998  0.001
 151 A H   0.006  0.996  0.001
 152 L H   0.022  0.990  0.001
 153 V H   0.067  0.961  0.002
 154 T H   0.181  0.863  0.020
 155 D H   0.378  0.622  0.029
 156 W H   0.269  0.811  0.006
 157 A H   0.258  0.809  0.012
 158 A H   0.299  0.731  0.012
 159 S H   0.373  0.661  0.009
 160 A C   0.516  0.466  0.018
 161 S C   0.728  0.219  0.030
 162 S C   0.789  0.205  0.013
 163 A C   0.554  0.487  0.008
 164 G C   0.508  0.457  0.008
 165 T H   0.153  0.874  0.002
 166 A H   0.036  0.977  0.000
 167 E H   0.022  0.976  0.000
 168 I H   0.014  0.983  0.000
 169 F H   0.015  0.984  0.000
 170 D H   0.016  0.987  0.000
 171 T H   0.016  0.988  0.000
 172 L H   0.015  0.991  0.000
 173 R H   0.011  0.993  0.000
 174 V H   0.010  0.994  0.000
 175 L H   0.009  0.994  0.000
 176 T H   0.008  0.994  0.000
 177 V H   0.008  0.993  0.000
 178 V H   0.010  0.993  0.000
 179 L H   0.010  0.993  0.000
 180 W H   0.011  0.992  0.000
 181 L H   0.013  0.992  0.000
 182 G H   0.012  0.994  0.000
 183 Y H   0.013  0.992  0.000
 184 P H   0.036  0.978  0.001
 185 I H   0.018  0.988  0.001
 186 V H   0.014  0.984  0.002
 187 W H   0.022  0.957  0.017
 188 A H   0.034  0.931  0.033
 189 V H   0.111  0.756  0.145
 190 G C   0.621  0.242  0.064
 191 V C   0.595  0.511  0.030
 192 E C   0.735  0.302  0.024
 193 G C   0.777  0.268  0.012
 194 L C   0.805  0.198  0.020
 195 A C   0.650  0.244  0.097
 196 L C   0.744  0.193  0.099
 197 V C   0.686  0.211  0.160
 198 Q C   0.754  0.141  0.095
 199 S C   0.826  0.145  0.024
 200 V H   0.490  0.629  0.010
 201 G H   0.241  0.807  0.024
 202 A H   0.084  0.910  0.033
 203 T H   0.009  0.992  0.001
 204 S H   0.004  0.994  0.001
 205 W H   0.004  0.988  0.001
 206 A H   0.008  0.985  0.000
 207 Y H   0.010  0.989  0.000
 208 S H   0.014  0.988  0.000
 209 V H   0.016  0.988  0.000
 210 L H   0.014  0.992  0.000
 211 D H   0.021  0.988  0.000
 212 V H   0.021  0.988  0.000
 213 F H   0.028  0.984  0.000
 214 A H   0.051  0.967  0.000
 215 K H   0.031  0.982  0.000
 216 Y H   0.014  0.994  0.000
 217 V H   0.007  0.997  0.000
 218 F H   0.004  0.998  0.000
 219 A H   0.003  0.998  0.000
 220 F H   0.002  0.998  0.000
 221 I H   0.002  0.997  0.000
 222 L H   0.004  0.995  0.001
 223 L H   0.007  0.994  0.001
 224 R H   0.011  0.993  0.000
 225 W H   0.022  0.988  0.000
 226 V H   0.164  0.867  0.002
 227 A H   0.369  0.661  0.008
 228 N C   0.533  0.498  0.013
 229 N C   0.765  0.263  0.007
 230 E C   0.678  0.328  0.006
 231 R C   0.648  0.373  0.008
 232 T C   0.791  0.214  0.017
 233 V C   0.824  0.150  0.022
 234 A C   0.912  0.055  0.022
 235 V C   0.979  0.000  0.002
