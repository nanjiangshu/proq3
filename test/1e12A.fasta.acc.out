1e12A_fasta_fastaalg
LLSSSLWVNVALAGIAILVFVYMGRTIRPGRPRLIWGATLMIPLVSISSYLGLLSGLTVGMIEMPAGHALAGEMVRSQWGRYLTWALSTPMILLALGLLADVDLGSLFTVIAADIGMCVTGLAAAMTTSALLFRWAFYAISCAFFVVVLSALVTDWAASASSAGTAEIFDTLRVLTVVLWLGYPIVWAVGVEGLALVQSVGATSWAYSVLDVFAKYVFAFILLRWVANNERTVAV
ebeebbeebbeebeebbeebeebbeebeeeebeebbebbbebbebbbebbebbeeeebbeebebbeebebeeeebebbebbbebbeebbbbbbebbbebbebeeebbeebbebbebbbebbbbbbebeeebeebeeebeebeeebeebbebbeeebbebbeebebbebbeebbeebbebbebbbebbbebbbbebeeeebbebbeebbebbbebbbebbebbbebbeeeeeebbe
